codus/codelog
# Auto bump

## Objective
- Add the `bump` command to automatically increment versions
- Resolve issue https://github.com/codus/codelog/issues/40.
## Notes
- `Gem::Version` documentation: http://ruby-doc.org/stdlib-2.0.0/libdoc/rubygems/rdoc/Gem/Version.html, _where you can see that it implements the 'spaceship' operator, making possible to compare instances of this class which are created passing the version string. This is used at the `last_created_changelog` method on the newly created `Bump` class._

## How to test
1. Add a new changelog
2. Run on the terminal `codelog bump [major|minor|patch]`
3. Check if the correct version is being released

## Checklist



[CONTRIBUTING]: https://github.com/codus/codelog/blob/master/CONTRIBUTING.md
